script.module.enum34/lib/enum/MANIFEST.in

```diff
@@ -0,0 +1,9 @@
+exclude enum/*
```

Comment:
remove

Reply:
This file was provided by the original author Ethan Furman.
This package is included in Python 3.4+ Versions.
Will removing this break the enum package?

Reply:
no this is only needed to packages for pypi

Reply:
OK